pulp_smash/tests/pulp3/constants.py

```diff
@@ -5,12 +5,16 @@
 
 BASE_PATH = '/api/v3/'
 
+BASE_CONTENT = urljoin(BASE_PATH, 'content/')
+
 BASE_IMPORTER_PATH = urljoin(BASE_PATH, 'importers/')
 
 BASE_PUBLISHER_PATH = urljoin(BASE_PATH, 'publishers/')
 
 DISTRIBUTION_PATH = urljoin(BASE_PATH, 'distributions/')
 
+FILE_CONTENT = urljoin(BASE_CONTENT, 'file/')
```

Comment:
Please name this `CONTENT_FILE_PATH` or `FILE_CONTENT_PATH`, and sort accordingly.